### ISSUE TITLE
Now windows phone support?

### ISSUE THREAD

**Issue by kimeuichan** [MobileChromeApps/cordova-plugin-zip#61]
my cordova-zip package.json

{
  "name": "cordova-plugin-zip",
  "version": "3.1.0",
  "description": "Unzips zip files",
  "cordova": {
    "id": "cordova-plugin-unzip",
    "platforms": [
      "android",
      "ios"
    ]
  },
  "repository": {
    "type": "git",
    "url": "https://github.com/MobileChromeApps/cordova-plugin-zip.git"
  },
  "keywords": [
    "ecosystem:cordova",
    "cordova-android",
    "cordova-ios"
  ],
  "engines": [
    {
      "name": "cordova",
      "version": ">=3.3.0"
    }
  ],
  "author": "",
  "license": "BSD",
  "bugs": {
    "url": "https://github.com/MobileChromeApps/zip/issues"
  },
  "homepage": "https://github.com/MobileChromeApps/zip"
}

i can make window 10 app with using this version?

**Comment by osro** [comment]
Seems like the issue is still open and the Windows sources are not yet merger into the master branch.

**Comment by rdbhagat999** [comment]
It is now 2019, and it seems that they haven't merged #60 into the master.

**Comment by osro** [comment]
> It is now 2019, and it seems that they haven't merged #60 into the master.

Yep, it is 2019 and everyone knows that Microsoft does not develop Windows Phone platform anymore. So there is no reason to merge it 😄

**Comment by rdbhagat999** [comment]
> > It is now 2019, and it seems that they haven't merged #60 into the master.
> 
> Yep, it is 2019 and everyone knows that Microsoft does not develop Windows Phone platform anymore. So there is no reason to merge it 😄

Well, I need to unzip a file for Windows 10 Universal app and I'm using the following plugin that currently supports only IOS and Android platforms.
